### DISCUSSION TITLE
[suggestion] warp jammer

### DISCUSSION THREAD

**Discussion by luckyrada** [tsunamayo/Starship-EVO#4020] [Suggestions]
### Short Description of the issue:

add a block that basically prevents any ship besides the ship its on from jumping into lightcruise / warping to another system or pulls them out of lightcruise when the block is turned on. there could be a set distance from the block itself (having a maximum distance you cant set it past obviously). now this would require a lot of energy to function and even more the farther the distance it jams, meaning ships like fighters would be incapable of using this block, and ships like flagships or big frigates would be able to. this would be awesome for a bunch of battle scenarios, making you plan your routes strategically at times in case of a jamming ship.

**Comment by Weskerdoo** [comment]
Name for block would be an Interdictor.

**Comment by luckyrada** [comment]
yes. thats great

**Comment by tsunamayo** [comment]
Yep I think we will need that.

**Comment by cloudrambler** [comment]
This method is a key mechanic in Elite Dangerous. That one mechanic was implemented very poorly by Frontier Developments and is the biggest problem with the game, and why it is close to unplayable for anyone who is not obsessed with PvP combat. It ruins an otherwise beautiful game, as anyone who is out for a fight has an immediate and enormous advantage over casual players. The game is very different to the classic Frontier which was so much better for play balance.

@ Tsuna, I am all for interesting mechanics and the more equipment/gizmos/junk I can load into my ships the better. But please exercise caution with warp jammers, as they can ruin the gameplay for those that want to see other players but don't always feel like fighting them. At the very least, there must be mechanics like random ship locations when they drop from warp, and huge cooldowns for the aggressive ship etc. Otherwise it's a gun in a knife fighting game.

**Comment by tsunamayo** [comment]
@cloudrambler , yep but Elite is more MMO oriented, and I am thinking more about single player balancing.
